doc/user_guide/tutorial.md

```diff
@@ -0,0 +1,472 @@
+
+```sql
+IMPORT INTO CHICAGO_TAXI_STAGE.TRIPS
+FROM CSV AT 'https://data.cityofchicago.org/resource' FILE 'wrvz-psew.csv?$limit=1000000'
+SKIP=1;
+```
+
+As mentioned earlier we use a slightly different API here, one that allows us to limit the number of datasets to 1 million. This way the import does not take forever.
+
+## Importing the Data Into Production
+
+First copy the data of the dimension tables.
+
+```sql
+INSERT INTO CHICAGO_TAXI.CENSUS_TRACTS
+SELECT * FROM CHICAGO_TAXI_STAGE.CENSUS_TRACTS;
+
+INSERT INTO CHICAGO_TAXI.COMMUNITY_AREAS
+SELECT * FROM CHICAGO_TAXI_STAGE.COMMUNITY_AREAS;
+```
+
+Use the following command to copy the trips data from the stage to production. Note how we additionally populate the extra column `EXA_ROW_TENANTS` with an ID.
+
+```sql
+INSERT INTO CHICAGO_TAXI.TRIPS
+SELECT TR.*,
+UPPER(
+    REGEXP_REPLACE(
+        REGEXP_REPLACE(
+            REGEXP_REPLACE(TR.COMPANY, '^[-0-9 ]*'),
+        '\W+', '_'
+        ),
+        '_*$', ''
+    )
+)
+FROM CHICAGO_TAXI_STAGE.V_TRIPS TR;
+```
+
+Note that this time we are copying the data from a view.
+
+The ID is an upper-case, underscore-connected version of the company name. Leading numbers and trailing underscores are removed.
+
+Let's take a look at the IDs this creates.
+
+```sql
+SELECT DISTINCT(COMPANY), EXA_ROW_TENANT
+FROM CHICAGO_TAXI.TRIPS
+ORDER BY EXA_ROW_TENANT;
+```
+
+| COMPANY                      | EXA_ROW_TENANT |
+|------------------------------|----------------|
+| 4053 - 40193 Adwar H. Nikola | ADWAR_H_NIKOLA |
+| 5074 - 54002 Ahzmi Inc       | AHZMI_INC      |
+| &hellip;                     | &hellip;       |
+| Yellow Cab                   | YELLOW_CAB     |
+| &hellip;                     | &hellip;       |
+| 3141 - 87803 Zip Cab         | ZIP_CAB        |
+| 3141 - Zip Cab               | ZIP_CAB        |
+| &hellip;                     | &hellip;       |
+| 5129 - 87128                 | `NULL`         |
+
+As you can see, the mapping is not perfect. There are two rows that produce the same ID (`ZIP_CAB`) and one that produces a `NULL`.
+
+In a real-world scenario you would spend more effort to disambiguate the ID generation and make sure that `NULL`s don't happen. For the sake of this tutorial though the naive approach will have to suffice.
+
+## Protecting the Data with RLS
+
+### Creating the RLS Virtual Schema
+
+Exasol's row-level security implementation is a [Virtual Schema](https://github.com/exasol/virtual-schemas). If you know how views work, then consider Virtual Schemas a closely related concept.
+
+A Virtual Schema is a projection of an underlying concrete schema. In the case of RLS it adds a filter layer that makes sure that users only see what they are supposed to.
+
+Please refer to the [user guide](user_guide.md#creating-the-virtual-schema) for detailed instructions on how to install the RLS package.
+
+```sql
+CREATE OR REPLACE JAVA ADAPTER SCRIPT RLS_VSADAPTER_SCHEMA.RLS_VSADAPTER AS
```

Comment:
Maybe we can use one of the existing schemas or add a create schema statement here for a users' convenience?

Reply:
I added the creation of the missing schema.